setuptools/_vendor/autocommand-2.2.2.dist-info/LICENSE

```diff
@@ -0,0 +1,166 @@
+GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Sorry to ghost post on you, but why did autocommand get added here? The rest look they were modifications, but this seems like a new _vendor package entirely. I ask because as it is LGPL, we have some licensing concerns. Thanks!

Reply:
Licensing concerns aside, I can answer your first question: `autocommand` is a dependency of `jaraco.text`. Setuptools doesn't need it, it's just pulled in with dependencies. If you *really* need to I'm sure you can devendor it.